monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Form from 'react-jsonschema-form';
 import {Col, Modal, Nav, NavItem} from 'react-bootstrap';
-import fileDownload from 'js-file-download';
+import FileSaver from "file-saver";
```

Comment:
Please use `'file-saver'` and not `"file-saver"` for consistency.

Reply:
Done